src/lib/pubkey/frodokem/frodo_constants.h

```diff
@@ -0,0 +1,105 @@
+/*
+* FrodoKEM constants
+* Based on the MIT licensed reference implementation by the designers
+* (https://github.com/microsoft/PQCrypto-LWEKE/tree/master/src)
+*
+* The Fellowship of the FrodoKEM:
+* (C) 2023 Jack Lloyd
+*     2023 René Meusel, Amos Treiber - Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_FRODOKEM_CONSTANTS_H_
+#define BOTAN_FRODOKEM_CONSTANTS_H_
+
+#include <botan/frodo_mode.h>
+#include <botan/internal/frodo_types.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace Botan {
+
+class XOF;
+
+class BOTAN_TEST_API FrodoKEMConstants {
+   public:
+      FrodoKEMConstants(FrodoKEMMode mode);
+
+      ~FrodoKEMConstants();
+
+      FrodoKEMConstants(const FrodoKEMConstants& other) : FrodoKEMConstants(other.m_mode) {}
+
+      FrodoKEMConstants(FrodoKEMConstants&& other) = default;
+      FrodoKEMConstants& operator=(const FrodoKEMConstants& other) = delete;
```

Comment:
`m_shake_xof` has a deleted copy assignment, so the default does not work and we'd have to build it ourselves. Since we only use the copy constructor, we just deleted it.